### PR TITLE
Doc: Logstash ingest pipelines

### PR DESCRIPTION
**Pull request by karenzone**
Related: 
* Issue #15172

Adds initial conceptual and "how to" info for Logstash Ingest pipelines. This doc is the conceptual counterpart to the `logstash-filter-elastic_integration` docs that focus on configuration and settings. 

**PREVIEW:**  https://logstash_15225.docs-preview.app.elstc.co/guide/en/logstash/master/ls-ingest-pipeline.html

### CODE REVIEW

File: docs/static/ls-ingest-pipeline.asciidoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+{ls} ingest pipeline functionality is implemented as a filter plugin, and is easy to integrate into new or existing {ls} pipeline configurations. 
+Add and configure the logstash-filter-elastic_integration filter as the _first_ filter in your {ls} pipeline. 
+
+// ToDo: Add code sample
+// ToDo: Add link to published plugin for settings and additional instructions.
+// For now: https://github.com/elastic/logstash-filter-elastic_integration
+
+Can set at the logstash or pipeline level 
+
+As with other plugins, you can set up ingest pipelines at the {ls} level in {logstash-ref}/logstash-settings-file.html[`logstash.yml`], or at the pipeline level in {logstash-ref}/pipelines.yml[`pipelines.yml`].
```
Comment (karenzone) [inline docs/static/ls-ingest-pipeline.asciidoc:32]:
```suggestion
As with other plugins, you can set up ingest pipelines at the {ls} level in {logstash-ref}/logstash-settings-file.html[`logstash.yml`], or at the pipeline level in {logstash-ref}/logstash-settings-file.html[`pipelines.yml`]. 
```

File: docs/static/ls-ingest-pipeline.asciidoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+[subs="attributes"]
+++++
+<titleabbrev>Add {ls} ingest pipeline processing</titleabbrev>
+++++
+
+{ls} ingest pipeline functionality is implemented as a filter plugin, and is easy to integrate into new or existing {ls} pipeline configurations. 
+Add and configure the logstash-filter-elastic_integration filter as the _first_ filter in your {ls} pipeline. 
+
+// ToDo: Add code sample
+// ToDo: Add link to published plugin for settings and additional instructions.
```
Comment (karenzone) [inline docs/static/ls-ingest-pipeline.asciidoc:27]:
```suggestion
// ToDo: Add link to plugin doc for settings and additional instructions after those docs are published.
```

**Comment by karenzone** [comment]
Hi @yaauie. Here's the initial conceptual content based on your EAH presentation and our Q&A session. (Thank you!) Please take a look and let me know what you think.

**Comment by karenzone** [comment]
@yaauie, will you please take a look at what we have so far to be sure we're on the right track and to determine what else is needed?

**Comment by karenzone** [comment]
Hi @flexitrev.  Here's the PR we discussed earlier. Looking forward to your feedback.

**Review by flexitrev — CHANGES_REQUESTED** [review]
We should overhaul what we are saying about this in the big picture. Logstash running ingest pipelines is a huge capability, but our primary use case is supporting Integrations through Logstash. Customers will certainly see the potential of this, and push it to the limit, but in our documentation we want to steer them to integrations because of the superior user experience.

**Comment by karenzone** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by karenzone** [comment]
Replaced by #15518

**Pull request closed without merging**